Bixal/move.mil
# Feature/156844285/parse zip5 file

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Address pivotal ticket #156844285
- Writes zip5.json

## Testing

To verify the changes proposed in this pull request…

1. Execute `docker-compose run php drupal parser` with zip5 option.
1. Verify zip5.json is written at `web/sites/default/files/tools/data`.
